src/roman_datamodels/testing/factories.py

```diff
@@ -13,6 +13,7 @@
 import numpy as np
 
 from .. import stnode
+from .. import tables
```

Comment:
Maybe we should rename tables.py to something a bit more descriptive or specific (just because "tables" is used in so many of our packages)? Maybe something like definitiion_tables.py? Not a hard issue, just a suggestion.

Reply:
What I have in mind for this module (eventually) is a method for each Roman table that creates an empty table object in the correct format.  We can test them against the schemas in this repo and then people who create reference files don't have to mess with dtypes.  Does that sound useful?

Reply:
Yes it does.

Reply:
Per conversation on Slack changing this module name to "table_definitions"